components/services/bat_ledger/bat_ledger_service_impl.cc

```diff
@@ -17,6 +17,10 @@ bool testing() {
   return ledger::is_testing;
 }
 
+bool debug() {
```

Comment:
Can it be a const method?

Reply:
I know two wrongs do not make a right, but I did not want to change too much in legacy code at this late stage so used same style as surrounding code

Reply:
Sure, that’s fine.